src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WebViewTests/UnoSamples_Test_NavigateToString.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.WebViewTests
+{
+	[TestFixture]
+	public partial class WebView_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void WebView_NavigateToLongString()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.WebView.WebView_NavigateToString2");
+
+			_app.WaitForElement(_app.Marked("startButton"));
+
+			TakeScreenshot("Initial");
+
+			var startButton = _app.Marked("startButton");
+			var clickResult = _app.Marked("WebView_NavigateToStringResult");
+
+			// step 1: generate long string
+			_app.Tap(startButton);
+
+			// _app.Wait(TimeSpan.FromSeconds(300)); // Android: 20 is sufficient
```

Comment:
Delete the commented code

Reply:
Commented code is left by purpose - as for WASM (which is interpreter, not compiler) default timeout can be too short, and maybe adding _Wait would be necessary for this.
But OK, I will delete it (but it means that I have to re-create PR, as I was told that it had to be single commit in PR, and I don't know how to make changes without adding new commit to PR)

Reply:
There's no need to recreate the PR. It's fine to add additional commits that address PR feedback.

Reply:
You shouldn't have multiple _extraneous_ commits - commit #1 'Fix X', commit #2 'X still not working, try something else', commit #3 'Fixed a typo' etc. There's no rule against multiple commits per se. 

The rule of thumb is that a commit should represent a single complete, logical unit of work.

Reply:
So, one commit per one changed file is ok?

Reply:
That's fine in this case.

Reply:
Also on the subject of commits - for the future please try to give each one a meaningful commit message. Don't just use the default 'Updated file.cs' when you're using the GitHub web interface.

Reply:
ok

Reply:
All corrections are made. 
So, can this PR be approved/merged? :)